awx/resource_workflow_job_template_schedule.go

```diff
@@ -106,3 +103,69 @@ func resourceWorkflowJobTemplateScheduleCreate(ctx context.Context, d *schema.Re
 	d.SetId(strconv.Itoa(result.ID))
 	return resourceScheduleRead(ctx, d, m)
 }
+
+func resourceWorkflowJobTemplateScheduleUpdate(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	var diags diag.Diagnostics
+	client := m.(*awx.AWX)
+	awxService := client.ScheduleService
+	id, diags := convertStateIDToNummeric("Update Schedule", d)
+	if diags.HasError() {
+		return diags
+	}
+
+	params := make(map[string]string)
+	_, err := awxService.GetByID(id, params)
+	if err != nil {
+		return buildDiagNotFoundFail("schedule", id, err)
+	}
+
+	_, err = awxService.Update(id, map[string]interface{}{
+		"name":                  d.Get("name").(string),
+		"rrule":                 d.Get("rrule").(string),
+		"workflow_job_template": d.Get("workflow_job_template_id").(int),
```

Comment:
From what I see in the doc https://docs.ansible.com/automation-controller/4.4/html/controllerapi/api_ref.html#/Schedules/Schedules_schedules_update
it's a unified_job_template in the PUT /api/v2/schedules/{id}.
Or maybe I'm looking a the wrong thing?

Reply:
Oops, sorry, my mistake.
I want to keep one of the `workflow_job_template_id` or `unified_job_template_id` parameters. Maybe just keep `unified_job_template_id`? In this case, there is no need to map the response parameter to the state object.